nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -865,7 +908,7 @@ define([
             }
             cellTabs.setState(state.tab);
```

Comment:
Ideally this code would also do something about ensuring that the current selected tab is valid, and if not, change to the default tab (see the next file).